releases/release-5.0.6.md

```diff
@@ -0,0 +1,169 @@
+---
+title: TiDB 5.0.6 Release Notes
+category: Releases
+---
+
+# TiDB 5.0.6 Release Notes
+
+Release date: December 28, 2021
+
+TiDB version: 5.0.6
+
+## Compatibility changes
+
+## Feature enhancements
+
+## Improvements
+
+## Bug fixes
+
+## __unsorted
+
++ PingCAP/TiDB
+
+    - This is a fix for grant global level case in v4.0 and v5.0 [#30161](https://github.com/pingcap/tidb/pull/30161)
+    - Fix an issue that adding index panics by chance. [#30125](https://github.com/pingcap/tidb/pull/30125)
+    - fix wrong result type for greatest/least [#29912](https://github.com/pingcap/tidb/pull/29912)
+    - Prevent conflicted optimistic transactions from locking each other. [#29776](https://github.com/pingcap/tidb/pull/29776)
+    - config: fix the bug that enforce-mpp *config item* is useless in v5.0.4 [#29637](https://github.com/pingcap/tidb/pull/29637)
+    - Fix panic for caseWhen function with enum type [#29509](https://github.com/pingcap/tidb/pull/29509)
+    - Fix wrong result of microsecond function in vectorized [#29385](https://github.com/pingcap/tidb/pull/29385)
+    - Fix incomplete log information about auto analyze. [#29228](https://github.com/pingcap/tidb/pull/29228)
+    - Fix wrong result of hour function in vectorized expression [#28871](https://github.com/pingcap/tidb/pull/28871)
+    - Fix unexpected error like `tidb_cast to Int32 is not supported` when unsupported cast is pushed down to TiFlash [#28651](https://github.com/pingcap/tidb/pull/28651)
+    - Fix bug that mpp node availability detect does not work in some corner cases [#28287](https://github.com/pingcap/tidb/pull/28287)
+    - sessionctx: fix data-race bug when alloc task id [#28284](https://github.com/pingcap/tidb/pull/28284)
+    - Fix index out of bound bug when empty dual table is remove for mpp query [#28278](https://github.com/pingcap/tidb/pull/28278)
+    - Avoid false positive error log about `invalid cop task execution summaries length` when running MPP query. [#28262](https://github.com/pingcap/tidb/pull/28262)
+    - Fixed a bug that causes TiDB panic when inserting invalid date value concurrently. [#28198](https://github.com/pingcap/tidb/pull/28198)
+    - Fix `can not found column in Schema column` error for mpp queries [#28147](https://github.com/pingcap/tidb/pull/28147)
+    - Fix a bug that TiDB may crash when TiFlash is shutting down. [#28138](https://github.com/pingcap/tidb/pull/28138)
+    - None. [#27927](https://github.com/pingcap/tidb/pull/27927)
+
++ TiKV/TiKV
```

Comment:
## Bug Fix

- Fix resolved timestamp lag increased after stoping a tikv [#11538](https://github.com/tikv/tikv/pull/11538)
- Fix connection abort caused by TiKV does not estimate the size of field `term` and index` [#11532](https://github.com/tikv/tikv/pull/11532)
- Fix panic in rare conditions when merge, conf change and snapshot happen at the same time [#11510](https://github.com/tikv/tikv/pull/11510)
- Fix the issue that reverse scan can't detect memory locks and may read stale data. [#11453](https://github.com/tikv/tikv/pull/11453)
- Add metrics for raft log garbage-collect to locate performance problem. [#11381](https://github.com/tikv/tikv/pull/11381)
- Fix negative sign when decimal divide to zero [#11333](https://github.com/tikv/tikv/pull/11333)
- Avoid possible OOM due to the accumulation of GC tasks [#11419](https://github.com/tikv/tikv/pull/11419)
-